README.md

```diff
@@ -1,18 +1,23 @@
 # LLVM Embedded Toolchain for Arm
 
-This repository contains build scripts and auxiliary material for building a bare-metal LLVM based toolchain targeting Arm based on:
+This repository contains build scripts and auxiliary material for building a
+bare-metal LLVM based toolchain targeting Arm based on:
 * clang + llvm
 * lld
 * libc++abi
 * libc++
 * compiler-rt
 * newlib
 
-Currently, only Armv6-M can be targeted, and only one library variant is provided targeting the Armv6-M architecture without any optional features.
+Currently, only Armv6-M can be targeted, and only one library variant is
+provided targeting the Armv6-M architecture without any optional features.
 
 ## Goal
 
-The goal is to provide a LLVM based bare-metal toolchain that can target the Arm architecture family from Armv6-M and newer. The toolchain follows the ABI for the Arm Architectue and attempts to provide typical features needed for embedded and realtime operating systems.
+The goal is to provide a LLVM based bare-metal toolchain that can target the
```

Comment:
My Google search told that it should be "a**n** LLVM". But please double check as well.